CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-06-14
```

Comment:
in my understanding 0.0.1 is not valid semver and the lowest you're supposed to go is `0.1.0`. I think it will work either way though